std/range/package.d

```diff
@@ -8269,6 +8269,22 @@ unittest // issue 14373
     assert(r.empty);
 }
 
+unittest // issue 14575
+{
+    struct R
+    {
+        Object front;
+        alias back = front;
+        bool empty = false;
+        void popFront() {empty = true;}
+        alias popBack = popFront;
+        @property R save() {return this;}
+    }
+    static assert(isBidirectionalRange!R);
+	R r;
+	auto rr = refRange(&r);
```

Comment:
Also, I'd suggest adding a second test which has Object as the element type but where `front` is a `const` function, since it's abnormal to have `front` be a public member variable, and the semantics of a public member variable and a property function aren't the same, and we don't want changes to the code to be committed which work in one case but not in the other.